RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectrons_cff.py

```diff
@@ -0,0 +1,55 @@
+import FWCore.ParameterSet.Config as cms
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronsPreRegression_cfi import lowPtGsfElectronsPreRegression
+
+lowPtGsfElectrons = lowPtGsfElectronsPreRegression.clone()
+
+################################################################################
+# LowPtGsfElectronProducer above is run by default in RECO
+# LowPtGsfElectronFinalizer below is scheduled for bParking and run2_miniAOD_devel
```

Comment:
the comment doesn't appear correct anymore.

Reply:
Updated.

Reply:
Updated.